docs/pages/_app.js

```diff
@@ -370,3 +370,13 @@ MyApp.getInitialProps = async ({ ctx, Component }) => {
     },
   };
 };
+
+export function reportWebVitals({ id, name, label, value }) {
+  window.ga('send', 'event', {
```

Comment:
At our scale, I wonder if we shouldn't throttle the events with the same logic as 

https://github.com/mui-org/material-ui/blob/fdd859c7e5d0007c3508012e0e4f93e6d12ffa1f/docs/src/modules/components/Ad.js#L210

There is at least a limit at 10,000,000 hits per month: https://developers.google.com/analytics/devguides/collection/analyticsjs/limits-quotas. It seems that we are already above the limit:

 > Last 30 Days: 14,122,534 hits

https://www.bounteous.com/insights/2015/11/10/google-analytics-10m-hit-limit-what-are-my-options/

Reply:
Reducing the number of hits for the demos could be one way to sort in the near future

<img width="688" alt="Capture d’écran 2020-07-07 à 17 11 02" src="https://user-images.githubusercontent.com/3165635/86801989-dde91200-c074-11ea-9054-b301f5f3d502.png">

Reply:
I don't know how often it is send. Since this is per property it's harmless to put it in, rightß

Reply:
> This function is fired when the final values for any of the metrics have finished calculating on the page. You can use to log any of the results to the console or send to a particular endpoint.

From this part of the documentation, it seems that the event is fired 8 times for the initial page load and 2 times per client-side page changes.

I would propose a 1% send ratio to see how that goes, as a first step. What do you think?

Reply:
I'll go over the Ad tomorrow. It would be nice if we could make this a two-way conversation:

> Since this is per property it's harmless to put it in, right?

> @oliviertassinari Do we send analytics events for deploy previews and branch builds?

Reply:
> Since this is per property it's harmless to put it in, right?

We have a single property.

>  Do we send analytics events for deploy previews and branch builds?

Yes, but the traffic on these live versions is negligible.

Reply:
> Yes, but the traffic on these live versions is negligible.

I was asking to see if I can test that it is working. Which property would receive events for this PR? I couldn't find `Web vitals` in the `development` property.

Reply:
I think that development is great if you want to try it out, before merging. If we want to merge, I think that we should deploy it on the master branch first. To battle test it.

Reply:
I think I wasn't clear before: Do we track deploy previews (e.g. https://deploy-preview-21702--material-ui.netlify.app/) and where? Do we track branch deploys (e.g. https://next--material-ui.netlify.app/) and where?

Reply:
I see the production tracking property id for these two URLs: UA-106598593-2.

Reply:
Shouldn't we use a different property for material-ui.netlify.app? Otherwise we're polluting real word traffic with development traffic.

Reply:
We could use a different property, the development one for instance. I think that the reason for not doing such so far is that there is a lot more traffic on the main property that it's a drop in the bucket. But for development purposes, this can help reduce noise.

Reply:
My point was more about production traffic being the noisy when I'm debugging analytics. So far I haven't seen a web vitals event so not sure what's happening.

Reply:
Do you have an ad blockers?

Reply:
I may or may not have flipped a numeric comparison :upside_down_face:

Reply:
Going with 0.01% for now. We can always scale this up later.  I'm also now filtering sessions so that we can track how metrics relate to each other per session.